packages/blade/src/components/Icons/SlashIcon/SlashIcon.tsx

```diff
@@ -0,0 +1,17 @@
+import type { IconComponent } from '..';
+import useIconProps from '../useIconProps';
+import { Path, Svg } from '../_Svg';
+
+const SlashIcon: IconComponent = ({ size, color }) => {
+  const { height, width, iconColor } = useIconProps({ size, color });
+  return (
+    <Svg height={height} width={width} viewBox="0 0 22 22" fill="none">
```

Comment:
Are these icons taken from the blade figma files? If so ideally they should all have same viewBox of `0 0 24 24` right? Make sure you are copying the correct layer. 

cc @saurav12 @Chaitanya-019

Reply:
I see, I copied the existing LinkIcon, and updated paths and variable names. That also has a viewBox os 0 0 22 22 apparently. Let me update it to `0 0 24 24`.

Reply:
@akkhil7 while copying the svg, you have to select the root icon element, in this case the parent "lock" element and not the ones within. All of these root svgs have a viewbox of `0 0 24 24`. After copying if the viewbox isn't `0 0 24 24` then there might be some issue while copying. Can you check this for all the icons added in this PR once?
<img width="352" alt="Screenshot 2022-11-22 at 3 05 04 PM" src="https://user-images.githubusercontent.com/24487274/203279145-1e0b0540-467d-4901-97c8-014f98b3cfca.png">